yarn.lock

```diff
@@ -914,6 +914,11 @@
   resolved "https://registry.yarnpkg.com/@types/tmp/-/tmp-0.2.3.tgz#908bfb113419fd6a42273674c00994d40902c165"
   integrity sha512-dDZH/tXzwjutnuk4UacGgFRwV+JSLaXL1ikvidfJprkb7L9Nx1njcRHHmi3Dsvt7pgqqTEeucQuOrWHPFgzVHA==
 
+"@types/uuid@^8.3.4":
```

Comment:
Unrelated. Updating yarn.lock to keep up with latest so that dependabot and Synk can give relevant information.